adapter/error.go

```diff
@@ -135,6 +135,12 @@ func ErrOAMRetry(err error) error {
 	return errors.New(ErrOAMRetryCode, errors.Alert, []string{"error marshal JSON: %s"}, []string{err.Error()}, []string{}, []string{})
 }
 
+//will be depracated
 func ErrGenerateComponents(err error) error {
 	return errors.New(ErrGenerateComponentsCode, errors.Alert, []string{"error generating components"}, []string{err.Error()}, []string{"Invalid component generation method passed, Some invalid field passed in DynamicComponentsConfig"}, []string{"Pass the correct GenerationMethod in DynamicComponentsConfig", "Pass the correct fields in DynamicComponentsConfig"})
 }
+
+//ErrCreatingComponents
+func ErrCreatingComponents(err error) error {
+	return errors.New(ErrCreatingComponentsCode, errors.Alert, []string{"error creating components"}, []string{err.Error()}, []string{"Invalid Path or version passed in static configuration"}, []string{"Make sure to pass correct configuration"})
```

Comment:
Think through with likely scenarios here are and try to provide guidance for our future selves. One more common issue could be that the location of a given service mesh's  helm charts have moved location and that location should be verified. Will you include this and other possible probable causes and suggested mediations?

Reply:
Made changes

Reply:
Thank you.